cli/worker.rs

```diff
@@ -381,13 +381,18 @@ pub async fn create_main_worker(
     create_web_worker_pre_execute_module_callback(ps.clone());
 
   let maybe_storage_key = ps.options.resolve_storage_key(&main_module);
-  let origin_storage_dir = maybe_storage_key.map(|key| {
+  let origin_storage_dir = maybe_storage_key.as_ref().map(|key| {
     ps.dir
       .root
       // TODO(@crowlKats): change to origin_data for 2.0
       .join("location_data")
       .join(checksum::gen(&[key.as_bytes()]))
   });
+  let cache_storage_dir = maybe_storage_key.map(|key| {
+    std::env::temp_dir()
+      .join("deno_cache")
```

Comment:
Why are you using a temporary directory? It will go away after the program finishes, isn't this cache supposed to be persistent?

Reply:
I didn't want to do quota management for the first iteration so I opted for tmp directory

Reply:
Please add a TODO for quota management.